### DISCUSSION TITLE
how do i get the weather remaster improved mod , Where do i go

### DISCUSSION THREAD

**Discussion by Funkster97** [CosmicHikari/Weather2-Remastered#83] [General]
how do i get the weather remaster improved mod , Where do i go

**Comment by CosmicHikari** [comment]
Go to the releases tab here and download it.

**Comment by CosmicHikari** [comment]
https://github.com/Mrbt0907/Weather2-Remastered/releases

**Reply by Funkster97** [reply]
i kinda need a step by step where to put it or extract it after download it  :/

**Comment by CosmicHikari** [comment]
I see. Go to files.minecraftforge.net and grab the latest version for
1.12.2.

Install forge. Once you've got forge installed go to your file explorer and
go to %appdata% . There's a folder called .Minecraft. go in there and
there's a mods folder. If the mods folder doesn't exist, create it and name
it mods.

Put weather2 remastered.jar in the mods folder. Then you need to download
coroutil 1.12.2 from curseforge. Make sure it's the 1.12.2 latest version.
Put it in mods too. Then your game is ready to play weather2 remastered. If
you need links let me know, I'm responding from my phone at the moment but
I can get the links to everything if you want.
